docs/guide/execution/quickrun_execution.rst

```diff
@@ -44,6 +44,69 @@ The ``quickrun`` command can be integrated into as:
 
   openfe quickrun transformation.json -o results.json
 
+Parallel execution of repeats with Quickrun
+===========================================
+
+Serial execution of multiple repeats of a transformation can be inefficient when working with a HPC, in this case higher
+throughput can be achieved by running one repeat per HPC job allowing for parallel execution. Most protocols are setup to
+run ``three repeats in seral`` by default, however, this can be changed either via the protocol setting ``protocol_repeats``, see the
```

Comment:
```suggestion
run ``three repeats in serial`` by default, however, this can be changed either via the protocol setting ``protocol_repeats``, see the
```